src/styles/card.scss

```diff
@@ -0,0 +1,116 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$hover-title-color: #34568b;
+
+.card {
+  height: 408px;
+  max-width: 200px;
+  transition-duration: 0.6s;
```

Comment:
The transition duration for the card is set to 0.6s, but the hover effect transition duration is set to 0.3s (line 105). Consider making these consistent to ensure uniform animation effects.